tests/functional/tests/security/test_management_start_fuzzy.py

```diff
@@ -154,3 +154,22 @@ def test_fuzzy_start_max_queue_size(pyocf_ctx, max_wb_queue_size, c_uint32_rando
         logger.warning(f"Test skipped for valid values: "
                        f"'max_queue_size={max_wb_queue_size}, "
                        f"queue_unblock_size={c_uint32_randomize}'.")
+
+
+@pytest.mark.security
+@pytest.mark.parametrize("cm", CacheMode)
+@pytest.mark.parametrize("cls", CacheLineSize)
+def test_fuzzy_start_promotion_policy(pyocf_ctx, c_uint32_randomize, cm, cls):
+    """
+    Test whether it is impossible to start cache with invalid promotion policy
+    :param pyocf_ctx: basic pyocf context fixture
+    :param c_uint32_randomize: promotion policy to start with
+    :param cm: cache mode value to start cache with
+    :param cls: cache line size to start cache with
+    """
+    if c_uint32_randomize not in [item.value for item in CacheLineSize]:
```

Comment:
I believe c_uint_32_randomize should be checked against PromotionPolicy enum instead of CacheLineSize

Reply:
Done